src/content/docs/errors-inbox/legacyerrors-eol.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Getting started with errors inbox'
+metaDescription: "Errors inbox is an error tracking solution designed to give you the tools to proactively detect, triage, and resolve errors across your full application stack."
+freshnessValidatedDate: never
+---
+
+We're excited to introduce a new and improved errors experience, designed to provide a more comprehensive and intuitive way to manage errors in your applications. To ensure a smooth transition, please take note of the following important updates:
+
+## Important Dates:
+
+* **January 31, 2024:** The legacy errors experience will be discontinued.
+* **Switch Now!** Access the new experience today by selecting the Show new view option.
+
+## Agent Updates Required
+
+Certain older agent versions do not produce `TransactionError` events, which are necessary for the new experience. To ensure full functionality, please update to the following minimum agent versions or higher:
+
+* Ruby – 3.14.0.305
+* PHP – 5.1.130
+* .NET – 6.8.172.0
+* Java – 3.21.0
+* Python – 2.58.0.43
+* Node.js – 1.24.0
+* Go - 0.5.0
+
+## Stack Trace Visibility
+
+In some cases, you may find that a stack trace is viewable in the legacy experience but not in the new one. This is due to a bookmarking feature that preserves certain stack traces beyond their standard retention period, but only if they were previously viewed in the legacy experience.
+
+To ensure access to these bookmarked stack traces in the new experience, please update your agent to the latest version.
+
+* Golang - 3.28.1
```

Comment:
You use dashes in the doc ahead of this, then switch to a colon. I actually think colons are clearer and better to use here/on the docs site